package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "angularx-flatpickr",
-  "version": "7.3.0",
+  "version": "8.0.0",
   "description": "An angular 14.0+ wrapper for flatpickr",
```

Comment:
Can you find and replace all references to `angular 14` -> `angular 15` in the repo?